wiki/Guides/Compressing_files/en.md

```diff
@@ -1,17 +1,31 @@
 # Compressing files
 
-Each beatmap has a file size limit dictated by its total length. This guide will help you to get your beatmap under that limit.
+Each beatmap has a file size limit dictated by it's total length and any video and audio content must meet requirements for formats, resolutions, and bit rates.
 
-There are 2 types of compression. **Lossy** and **Lossless** compression.
+This guide will help you get your beatmap under that limit and meet such requirements.
 
-- **Lossless** implies that the quality never degrades and can be repeatedly compressed or decompressed.
-- **Lossy** is a much more powerful form of compression which sacrifices quality for space/processor demand.
+## Introduction
+
+There are 2 types of compression. **Lossless** and **Lossy** compression.
+
+- **Lossless** compression implies that the quality never degrades and can thus be repeatedly compressed and decompressed.
+- **Lossy** compression uses certain powerful techniques to reduce file size at the expense of quality.
+
+To convert, or reduce either file size, average bitrate, or resolution of a audio or video file requires a process called **re-encoding**, or more known as **transcoding**. Transcoding an already lossy-compressed audio or video using lossy compression can result in varying degrees of further quality reduction depending on the settings used.
+
+Due to that reason, re-encoding should be avoided except if the source audio or video file is any of the following:
+
+- Too large in file size
+- Too high of a resolution or average bit rate
+- Encoded in a incompatible format
+
+If you have to re-encode, it is suggested to use the highest quality source file available (e.g. highest resolution and/or bit rate) from which to be re-encoded from.
```

Comment:
```suggestion
In case re-encoding is necessary, it is suggested to use the highest-quality source file available, i.e. with the highest resolution and/or bit rate.
```